WcaOnRails/env_vars.rb

```diff
@@ -8,7 +8,7 @@
     mandatory :SMTP_PASSWORD, string
   end
 
-  if Rails.env.development?
+  unless Rails.env.production?
```

Comment:
This change seems trivial enough, but I'm wondering whether you have any specific reason for including it in this PR?

Reply:
Yes, without it `EnvVars.AWS_ACCESS_KEY_ID` and `EnvVars.AWS_SECRET_ACCESS_KEY` are not declared and can't be used in the test environment; the only side effect is to include `DISABLE_BULLET` in the changes, but since I noticed the `.env.test` does define it I figured it was fine.

Reply:
Okay, thanks :)